mindjunkies/home/views.py

```
@@ -1,15 +1,54 @@
+
+    return popular_courses
+
+
+def proxy_pdf(request):
+    pdf_url = "https://blr1.digitaloceanspaces.com/mindjunkies/media/lecture_pdfs/Project_Proposal_for_swe_2.pdf"
+    response = requests.get(pdf_url)
+    return HttpResponse(response.content, content_type='application/pdf')
+
+
+def get_new_courses():
+    cache_key = f"new_courses"
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)